infrastructure/templates/template.json

```diff
@@ -564,9 +564,22 @@
       "metadata": {
         "description": "Slack webhook URI for alerts"
       }
+    },
+    "imageTemplateName": {
+      "type": "string"
+    },
+    "svcLocation": {
+      "type": "string"
+    },
+    "managedImageResGroupId": {
+      "type": "string"
+    },
+    "managedImageName": {
+      "type": "string"
```

Comment:
Any reason we have any changes in this file? `template.json` and `image-template.json` should be completely independent of one another so that we can deploy one without the other.

It wouldn't be a good idea to be have to rebuild the test runner image every time we want to deploy the rest of our infrastructure as building an image isn't necessarily fast.

Reply:
I think this must have been something leftover from when I force pushed during standup. Should be fixed now